bin/dev

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+
+if gem list --no-installed --exact --silent foreman; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
```

Comment:
Wouldn't this be expected to be installed through the `make setup` process?

https://github.com/18F/identity-dashboard/blob/52efcfcae6ff115355c0d64b8779b9a56a08e6d6/bin/setup#L44